substrate/bin/node/testing/src/keyring.rs

```diff
@@ -20,52 +20,51 @@
 
 use codec::Encode;
 use kitchensink_runtime::{CheckedExtrinsic, SessionKeys, SignedExtra, UncheckedExtrinsic};
-use node_cli::chain_spec::get_from_seed;
 use node_primitives::{AccountId, Balance, Nonce};
-use sp_core::{ecdsa, ed25519, sr25519};
+use sp_core::{crypto::get_public_from_string_or_panic, ecdsa, ed25519, sr25519};
 use sp_crypto_hashing::blake2_256;
-use sp_keyring::AccountKeyring;
+use sp_keyring::Sr25519Keyring;
 use sp_runtime::generic::Era;
 
 /// Alice's account id.
 pub fn alice() -> AccountId {
-	AccountKeyring::Alice.into()
+	Sr25519Keyring::Alice.into()
 }
 
 /// Bob's account id.
 pub fn bob() -> AccountId {
-	AccountKeyring::Bob.into()
+	Sr25519Keyring::Bob.into()
 }
 
 /// Charlie's account id.
 pub fn charlie() -> AccountId {
-	AccountKeyring::Charlie.into()
+	Sr25519Keyring::Charlie.into()
 }
 
 /// Dave's account id.
 pub fn dave() -> AccountId {
-	AccountKeyring::Dave.into()
+	Sr25519Keyring::Dave.into()
 }
 
 /// Eve's account id.
 pub fn eve() -> AccountId {
-	AccountKeyring::Eve.into()
+	Sr25519Keyring::Eve.into()
 }
 
 /// Ferdie's account id.
 pub fn ferdie() -> AccountId {
-	AccountKeyring::Ferdie.into()
+	Sr25519Keyring::Ferdie.into()
 }
 
 /// Convert keyrings into `SessionKeys`.
```

Comment:
I suggest to add that in the doc a `#panic` section sayin that if `seed` is invalid this panics